lib/winapi/.clang-format

```diff
@@ -0,0 +1,21 @@
+BasedOnStyle: Microsoft
+ColumnLimit: 0
+SortIncludes: false
```

Comment:
We might actually want this. The way I prefer it is to separate my include statements into groups, and have each group alphabetically sorted, but the groups as a whole not reordered.

Reply:
ill try it, not a fan tbh as not all includes behave nicely and ive had cases of code not compiling after running clang-format :P

If it doesnt break anything ill add it